identifier/definitionmanager.cpp

```diff
@@ -258,6 +266,13 @@ void DefinitionManager::installJson(QString path, bool overwrite,
   QString key = def->at("name")->asString() + def->at("type")->asString();
   QString exeversion = def->at("version")->asString();
 
+  // check if intermediate qHash(..,0) name is present
+  QString dest0 = destdir + "/" + QString("%1").arg(qHash(key, 0)) + ".json";
+  if (QFile::exists(dest0)) {
+    QFile file (dest0);
+    file.remove();
+  }
+
```

Comment:
I think this should be more documented. At least a link to the issue.

Reply:
My decision was: "It is save to delete definition files"
In the past we had already a list with hard coded hashed names we always delete, this is extending this list.